x-pack/test/api_integration/apis/metrics_ui/metrics_overview_top.ts

```diff
@@ -49,5 +48,47 @@ export default function ({ getService }: FtrProviderContext) {
       expect(series[0].id).to.be('demo-stack-mysql-01');
       expect(series[0].timeseries[1].timestamp - series[0].timeseries[0].timestamp).to.be(300_000);
     });
+
+    describe('Runtime fields calculation', () => {
+      before(() =>
+        esArchiver.load('x-pack/test/functional/es_archives/infra/8.0.0/hosts_and_network')
+      );
+      after(() =>
+        esArchiver.unload('x-pack/test/functional/es_archives/infra/8.0.0/hosts_and_network')
+      );
+
+      it('should return correct calculations', async () => {
+        const { min, max } = DATES['8.0.0'].hosts_and_netowrk;
+        const response = await supertest
+          .post('/api/metrics/overview/top')
+          .set({
+            'kbn-xsrf': 'some-xsrf-token',
+          })
+          .send(
+            TopNodesRequestRT.encode({
+              sourceId: 'default',
+              bucketSize: '300s',
+              size: 5,
+              timerange: {
+                from: min,
+                to: max,
+              },
+            })
```

Comment:
Could we validate sorting by `rx` or `tx` too? I guess with that we'll make sure the sorting will always work for those fields

```suggestion
            TopNodesRequestRT.encode({
              sourceId: 'default',
              bucketSize: '300s',
              size: 5,
              timerange: {
                from: min,
                to: max,
              },
            },
            sort: 'rx', 
            sortDirection: 'desc' })
```

Reply:
Good Catch, I agree let me put it to this PR and then we can merge it all at once